### ISSUE TITLE
UI Corruption starting on jupyter/scipy-notebook:64c893e91c60 after jupyter lab build?

### ISSUE THREAD

**Issue by kronenpj** [jupyter/docker-stacks#934]
**What docker image you are using?**
`jupyter/scipy-notebook:64c893e91c60` and later, including `:latest` as of today (6-Sep-2019).

**What complete docker command do you run to launch the container (omitting sensitive values)?**
```bash
docker run -d -p 9999:8888 --rm \
     --name test-notebook \
     -e JUPYTER_ENABLE_LAB=yes \
     jupyter/scipy-notebook:64c893e91c60 \
     start-notebook.sh --LabApp.token=''
```

**What steps do you take once the container is running to reproduce the issue?**
1. Open a temrinal within the UI
2. Run: `jupyter lab build`
3. Reload the UI in the browser (reload button)

Also occurs with the following `Dockerfile` when running the resulting image immediately:
```
FROM jupyter/scipy-notebook:64c893e91c60
USER $NB_UID
RUN jupyter lab build
```

**What do you expect to happen?**
The UI remains essentially the same.

**What actually happens?**
The user interface elements along the left-hand side (folders, terminals/kernels, commands, etc. icons) don't disappear.
The elements at the bottom left don't disappear (# terminal sessions & kernels).
New tabs of almost any type (except Launcher) are corrupt.

I've investigated this and found that `jupyter/scipy-notebook:29105a731d24` and earlier don't seem to exhibit this behavior. Unfortunately I've so far been unsuccessful at identifying which specific change actually causes this behavior. The changes in `scipy-notebook/Dockerfile` don't appear to be the cause, so I'm moving into `minimal-` and `base-notebook`. 

I'm really beginning to wonder whether this is something only I'm seeing :)

**Comment by kronenpj** [comment]
I found that it doesn't happen when using `jupyter:minimal-notebook`.

**Comment by jeremydouglass** [comment]
> New tabs of almost any type (except Launcher) are corrupt.

What does "corrupt" mean? What are you seeing exactly?

**Comment by kronenpj** [comment]
Since I did my best in words, here's a picture...

![screenshot](https://user-images.githubusercontent.com/286689/64466264-91554300-d100-11e9-947f-59908242eeab.png)

**Comment by kronenpj** [comment]
Using `jupyter/minimal-notebook@64c893e91c60f60f9adb2a1e1acc934673a3388c`, I've been running through the `Dockerfile` line by line, executing those commands inside the container at the terminal window, then refreshing the page after each item. 
The UI is corrupted after I run `jupyter labextension install @jupyter-widgets/jupyterlab-manager@^1.0.0`. 

I then reset the environment to `jupyter/minimal-notebook@64c...` and ran `jupyter labextension install @jupyter-widgets/jupyterlab-manager@^1.0.0` from the terminal, same behavior.

It appears as though using `jupyter labextension` causes the problem. I've tried `jupyterlab-drawio`, 'jupyterlab_bokeh`, and `@jupyterlab/git`. Hardly a representative sample, I know. Installing them through the extension manager has the same effect.

As a last experiment, I reset again and tried only `jupyter lab build` from a terminal. That did _not_ exhibit the corruption.

**Comment by kronenpj** [comment]
Looks like this might have been fixed somewhere after jupyterlab v1.0.4 because when I build `base-notebook` with `jupyterlab-1.0.10` I don't see the behavior.

**Comment by kronenpj** [comment]
Verified my issue is fixed with the recent update of Jupyter and Jupyterlab.
